### PR TITLE
Log segger version string when connecting to a devkit

### PR DESCRIPTION
**Pull request by IvanSanchez**
Pretty much the same as https://github.com/NordicPlayground/pc-nrfconnect-programmer/pull/21

### CODE REVIEW

**Comment by bencefr** [comment]
I wonder if this also needs an update to package.json -> engines.nrfconnect, since this is a released app, we need to more careful about compatibility.

**Comment by IvanSanchez** [comment]
@chunfantasy Getting the probe info takes around 1.5 secs (anything from 1.3 to 1.8). I don't think @CoqRogue can speed this up any further.

Note that the logging the probe info only happens *once*, when connecting to the connectivity devkit, before the connectivity firmware is checked/flashed.

**Comment by chunfantasy** [comment]
image
Looks like it does not use the proper pc-nrfjprog-js.
I have tried `npm i` `npm update`

**Comment by bencefr** [comment]
The correct nrfjprog is defined by the host application nrfConnect, so you'll need to update that. This is also the reason why the engine needs to refer to upped version of core.

**Comment by IvanSanchez** [comment]
So which version of `pc-nrfconnect-core` includes `pc-nrfjprog-js` v1.2.0? I can't readily find that info in its [changelog](https://github.com/NordicSemiconductor/pc-nrfconnect-core/releases).

**Comment by bencefr** [comment]
Yet unreleased 2.3+

**Comment by chunfantasy** [comment]
Right, the master branch of core works fine.

**Comment by bencefr** [comment]
Try opening and closing explicitly as @CoqRogue suggested in order to speed up.

File: lib/actions/adapterActions.js [hunk 1]
```diff
@@ -1112,7 +1112,13 @@ export function selectedSerialPort(port) {
     return dispatch => (
         new Promise(() => {
             dispatch(closeAdapter(() => {
-                getDeviceInfo(port.serialNumber)
+                getProbeInfo(port.serialNumber)
+                    .then(probeInfo => {
+                        logger.info('Segger serial: ', probeInfo.serialNumber);
+                        logger.info('Segger speed: ', probeInfo.clockSpeedkHz, ' kHz');
```
Comment (bihanssen) [inline lib/actions/adapterActions.js]: I'd argue that segger clock speed is not relevant to show in the log view, rather show in log file only (logger.debug)

**Comment by mrodem** [comment]
Replaced by #81.

**Pull request closed without merging**